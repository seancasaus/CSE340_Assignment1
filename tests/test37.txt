FHTOy (((h)|((Z).(N)))|((U).(((k)|(i)).(1)))).((n).(L)) , V6r0K (((e)|(P))|((v)|((T)|(f)))).(0) , oDQP7 (((((s)|(E)).((y)|(u)))|((m)|(h))).(t))|(((O)|(1)).((g).(q))) , aH6A9 ((v).((o).((t).((t)|(P))))).((((Q).(4))|(m))|(I)) , NjOIF h , iUlGg 6 , tM ((((t)|(W)).(z)).((C)|((7).((8)|(R)))))|(x) , JRHuK (d)|(i) , Sg 2 , LU ((M).((((g)|(8)).((R)|(q)))|(((M)|(G))|(u))))|(((7).(s)).((((u)|(G)).(3)).((a).(T)))) #
" ZNnL7sG3aTe0vottmvottQ4hthnLUk1nLvottIdvottIOgq227sG3aTUi1nLxhhvottIZNnLht2ZNnLvottIZNnLhnL2sytde0EytvotPQ4vottIv56x2i66dtzC2Uk1nLvotPm66ZNnLvottm62mtWzCh267sG3aThWzChMgqvotPItzC2ZNnLMgRMuUi1nL2votPmhhtxvottmvotPIP0 " 