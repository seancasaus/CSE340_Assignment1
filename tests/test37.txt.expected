FHTOy , "ZNnL"
LU , "7sG3aT"
V6r0K , "e0"
aH6A9 , "vottm"
aH6A9 , "vottQ4"
oDQP7 , "ht"
FHTOy , "hnL"
FHTOy , "Uk1nL"
aH6A9 , "vottI"
JRHuK , "d"
aH6A9 , "vottI"
oDQP7 , "Ogq"
Sg , "2"
Sg , "2"
LU , "7sG3aT"
FHTOy , "Ui1nL"
tM , "x"
NjOIF , "h"
NjOIF , "h"
aH6A9 , "vottI"
FHTOy , "ZNnL"
oDQP7 , "ht"
Sg , "2"
FHTOy , "ZNnL"
aH6A9 , "vottI"
FHTOy , "ZNnL"
FHTOy , "hnL"
Sg , "2"
oDQP7 , "syt"
JRHuK , "d"
V6r0K , "e0"
oDQP7 , "Eyt"
aH6A9 , "votPQ4"
aH6A9 , "vottI"
ERROR
